apps/test/unit/lib/kits/maker/ui/EligibilityChecklistTest.js

```diff
@@ -146,7 +140,6 @@ describe('EligibilityChecklist', () => {
         {...defaultProps}
```

Comment:
This test was supposed to check the problem that this PR fixes, but was conditional on whether the user got the full discount (with shipping). Removed that condition from this test (and other tests).